### PR TITLE
[SR-8139] Fix executable init template when the package name has a dash

### PR DESCRIPTION
**Pull request by JhonnyBillM**

### CODE REVIEW

File: Tests/WorkspaceTests/InitTests.swift [hunk 1]
```diff
@@ -177,4 +177,25 @@ class InitTests: XCTestCase {
...
+        // Create the package
+        let initPackage = try InitPackage(name: "package-name", destinationPath: packageRoot, packageType: InitPackage.PackageType.executable)
+        initPackage.progressReporter = { message in
+        }
+        try initPackage.writePackageStructure()
+        
+        // Try testing it.
+        XCTAssertSwiftTest(packageRoot)
+        
+//        XCTAssertFileExists(packageRoot.appending(components: ".build", Destination.host.target, "debug", "some_package.swiftmodule"))
```
[the remark this answers is not in the document]
Comment (JhonnyBillM) [reply Tests/WorkspaceTests/InitTests.swift]: Sorry, quick copy/pasting and did not verify properly 😔

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Comment by aciidb0mb3r** [comment]
Yep, nice catch @rahul-malik. Can you open a PR with the fix?

**Comment by rahul-malik** [comment]
PR fix is here: 
https://github.com/apple/swift-package-manager/pull/1706

**Pull request merged**